x-pack/plugins/ml/common/constants/alerts.ts

```diff
@@ -20,8 +21,26 @@ export const TOP_N_BUCKETS_COUNT = 1;
 
 export const ALL_JOBS_SELECTION = '*';
 
-export const HEALTH_CHECK_NAMES = {
+export const HEALTH_CHECK_NAMES: Record<JobsHealthTests, string> = {
   datafeed: i18n.translate('xpack.ml.alertTypes.jobsHealthAlertingRule.datafeedCheckName', {
     defaultMessage: 'Datafeed is not started',
   }),
+  mml: i18n.translate('xpack.ml.alertTypes.jobsHealthAlertingRule.mmlCheckName', {
+    defaultMessage: 'Model memory limit',
+  }),
+  errorMessages: i18n.translate(
+    'xpack.ml.alertTypes.jobsHealthAlertingRule.errorMessagesCheckName',
+    {
+      defaultMessage: 'There are errors in job messages',
+    }
+  ),
+  behindRealtime: i18n.translate(
+    'xpack.ml.alertTypes.jobsHealthAlertingRule.behindRealtimeCheckName',
+    {
+      defaultMessage: 'Job is running behind real-time',
+    }
+  ),
+  delayedData: i18n.translate('xpack.ml.alertTypes.jobsHealthAlertingRule.delayedDataCheckName', {
+    defaultMessage: 'Delayed data has occurred',
```

Comment:
nit. I'd switch the word order here.
```suggestion
    defaultMessage: 'Data delay has occurred.',
```

Reply:
Updated in b73d27b7abebb21dd393529cca351d0af5eb43bc